spring-kafka/src/main/java/org/springframework/kafka/listener/MessageListenerContainer.java

```diff
@@ -39,11 +39,13 @@
 	 */
 	void setupMessageListener(Object messageListener);
 
-	/**
-	 * See {@link org.apache.kafka.clients.consumer.Consumer#metrics()}.
-	 * @return the consumer(s) metrics grouped by {@code client-id}
-	 * @since 2.0
-	 */
-	Map<String, Map<MetricName, Metric>> metrics();
+    /**
+     * Returns metrics for this container's consumer(s), grouped by {@code client-id}.
+     *
+     * @return the consumer(s) metrics grouped by {@code client-id}
+     * @see org.apache.kafka.clients.consumer.Consumer#metrics()
+     * @since 2.0
```

Comment:
Just nit-picking, but proper order that `@see` is the latest tag in the JavaDoc.
Also no blank lines in the method JavaDocs, please.
Plus the general sentence on the method JavaDoc should be in the imperative, 

The mentioned JavaDocs in the `KafkaOperations` must be fixed as well. Thanks. But that is already out of the scope your task.

Reply:
hmm, usually `@since`, `@serial` and `@deprecated` are after `@see`, but If you want it to be the last, I don't mind.

Reply:
Yeah...
This is our Checkstyle rule:
```
<module name="AtclauseOrder">
			<property name="target" value="METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
		<property name="tagOrder" value="@param, @return, @throws, @since, @deprecated, @see"/>
</module>
```
This is our code style: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#javadoc-formatting